repository/src/main/java/org/alfresco/transform/client/registry/CombinedConfig.java

```diff
@@ -273,277 +238,7 @@ public void register(TransformServiceRegistryImpl registry)
         data.setTEngineCount(tEngineCount);
         data.setFileCount(configFileFinder.getFileCount());
 
```

Comment:
The following code is now in the common CombinedTransformConfig class of the transform model for use by both the Repo and t-router.